### ISSUE TITLE
Hyperlinking of fields in record view

### ISSUE THREAD

**Issue by seanluyk** [ualbertalib/discovery#580]
I wonder if it might be useful to discuss which fields are planned to be hyperlinked in records views. I like that both subjects and authors are hyperlinked already, however, there are some more that might be useful too.

Some other useful ones from my perspective...
1. **Uniform title** (once issues with getting subfields to display are fixed)
2. **Title** - this is an iLink feature that's actually quite useful. For example, if you discover a record that turns out to be an ebook when you want a print book and vice versa, it's nice to not have to redo the search
3. **Any additional authors** (added-entry personal fields) - especially useful for edited works, collections, anthologies, etc.
4. **Series** - OK, this is probably most useful for librarians, but if it isn't lots of work to make this hyperlinked, it's well worth it. It comes in handy working with users

**Comment by redlibrarian** [comment]
Let's discuss this in the new year.

**Comment by redlibrarian** [comment]
@seanluyk OK. I've made the changes specified in the note (except additional authors, which I need to do separately). I've figured out how to customize the display of all the fields, which means I can get rid of a "find more by this author" section of the page, and just hyperlink the author field. I'll do this with additional authors as well. I'll let you know when this is in test.

**Comment by seanluyk** [comment]
@redlibrarian  - this is great, I'll relabel and self-assign so I remember to check in on this.

**Comment by seanluyk** [comment]
The fields in this issue appear to be hyperlinked, it looks/works great! I don't think it's quite ready to close, as there's an issue with series hyperlinking. For example:

1. [This one works as expected](https://search-test.library.ualberta.ca/catalog/5447754)

2. [This one only returns 1 result](https://search-test.library.ualberta.ca/catalog/1743283)

I wonder if it would be better to not search the volume number, just the series name?

**Comment by theLinkResolver** [comment]
Agreed - we want to see the series volume number displayed (and be indexed alongside the series title, as that is how it may be cited and/or how people search for it) but not part of the search string. (Or I guess in the perfect world, it would work like the subject headings - click the title or title+volume. Long-run/daydream stuff)

**Comment by redlibrarian** [comment]
@theLinkResolver Looking at the MARC record, the series name is 830$a and the number 830$v, is that right?

**Comment by theLinkResolver** [comment]
@redlibrarian Use the 830 $a.

**Comment by theLinkResolver** [comment]
@seanluyk The issue you described in this issue on Apr 21 is still a problem. I don't know why this was closed, actually (a few years ago). Here is another example of the problem, which I would characterize as a bug.

https://www.library.ualberta.ca/catalog/8011693

The expected function of clicking the hyperlinked series title (i.e. show me more of the series) should result in [this](https://www.library.ualberta.ca/symphony?utf8=%3F&q=%22Advances+in+Anatomy+Embryology+and+Cell+Biology%22) - a search of the title as a phrase, and without the volume number from the record you're looking at.

(And really, we should do this function using the 8XX series statements, as they use controlled vocabulary. The 490 is just a transcription of the series title from this particular item. But that's another discussion - within scope of #1146.)

I'm leaving this closed as you may wish to enter this in a fresh issue in the current context, as this one covered a bunch of things, or leave it aside pending future directions.

**Comment by weiweishi** [comment]
@seanluyk let's discuss this in our next grooming meeting re: detailed requirements, and how this is different from #1542 and how this is tied to some of the work in record view revamp task

**Comment by theLinkResolver** [comment]
@seanluyk Let me know if you are looking wholesale at what is and isn't hyperlinked - there are some things not mentioned above that we should probably have in mind (especially with respect to discovery replicating classic catalogue functions - if that's still an objective! I might be a few years out of date).
